tools/kbhit.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+"""https://simondlevy.academic.wlu.edu/files/software/kbhit.py.
+
+A Python class implementing KBHIT, the standard keyboard-interrupt poller.
+Works transparently on Windows and Posix (Linux, Mac OS X).  Doesn't work
+with IDLE.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of the
```

Comment:
I think the LGPL is requires you to dynamically link the module. IANAL, but for Python I think that would be telling users that they have to run `pip install kbhit` in order to use your library.
Since we're distributing this to partners this matters.